packages/augur-sdk/src/utils.ts

```diff
@@ -11,6 +11,12 @@ import { toAscii } from './state/utils/utils';
 
 export const QUINTILLION = new BigNumber(10).pow(18);
 
+export function marketNameToType(
+  marketName: string
+): number {
+  return MarketType[marketName];
```

Comment:
need to handle error conditions. like name isn't in list. prob just throw an error.